File: WEB/src/Entity/Programmealimentaire.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Programmealimentaire
 *
 * @ORM\Table(name="programmealimentaire")
 * @ORM\Entity
 */
class Programmealimentaire
{
    /**
     * @var string
     *
     * @ORM\Column(name="idProfile", type="string", length=20, nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idprofile;

    /**
     * @var int|null
     *
     * @ORM\Column(name="bmr", type="integer", nullable=true)
     */
    private $bmr;

    /**
     * @var int|null
     *
     * @ORM\Column(name="calRequis", type="integer", nullable=true)
     */
    private $calrequis;

    /**
     * @var int|null
     *
     * @ORM\Column(name="fatsRequis", type="integer", nullable=true)
     */
    private $fatsrequis;

    /**
     * @var int|null
     *
     * @ORM\Column(name="carbsRequis", type="integer", nullable=true)
     */
    private $carbsrequis;

    /**
     * @var int|null
     *
     * @ORM\Column(name="proteinsRequis", type="integer", nullable=true)
     */
    private $proteinsrequis;

    public function getIdprofile(): ?string
    {
        return $this->idprofile;
    }

    public function getBmr(): ?int
    {
        return $this->bmr;
    }

    public function setBmr(?int $bmr): self
    {
        $this->bmr = $bmr;

        return $this;
    }

    public function getCalrequis(): ?int
    {
        return $this->calrequis;
    }

    public function setCalrequis(?int $calrequis): self
    {
        $this->calrequis = $calrequis;

        return $this;
    }

    public function getFatsrequis(): ?int
    {
        return $this->fatsrequis;
    }

    public function setFatsrequis(?int $fatsrequis): self
    {
        $this->fatsrequis = $fatsrequis;

        return $this;
    }

    public function getCarbsrequis(): ?int
    {
        return $this->carbsrequis;
    }

    public function setCarbsrequis(?int $carbsrequis): self
    {
        $this->carbsrequis = $carbsrequis;

        return $this;
    }

    public function getProteinsrequis(): ?int
    {
        return $this->proteinsrequis;
    }

    public function setProteinsrequis(?int $proteinsrequis): self
    {
        $this->proteinsrequis = $proteinsrequis;

        return $this;
    }


}

File: Mobile/src/Entities/programmeAlimentaire.java
package Entities;

public class programmeAlimentaire{
    
    protected String idProfile;
    protected int bmr;
    protected int calRequis;
    protected int fatsRequis;
    protected int carbsRequis;
    protected int proteinsRequis;
    protected String emailProfile;
    
    public programmeAlimentaire(){
    }
       
    public programmeAlimentaire(String idProfile, int bmr, int calRequis, int fatsRequis, int carbsRequis, int proteinsRequis){
        this.idProfile = idProfile;
        this.bmr = bmr;
        this.calRequis = calRequis;
        this.fatsRequis = fatsRequis;
        this.carbsRequis = carbsRequis;
        this.proteinsRequis = proteinsRequis;
    }
    
    public programmeAlimentaire(String idProfile, int bmr, int calRequis, int fatsRequis, int carbsRequis, int proteinsRequis, String emailProfile){
        this.idProfile = idProfile;
        this.bmr = bmr;
        this.calRequis = calRequis;
        this.fatsRequis = fatsRequis;
        this.carbsRequis = carbsRequis;
        this.proteinsRequis = proteinsRequis;
        this.emailProfile = emailProfile;
    }
    
    public void setEmailProfile(String emailProfile){
            this.emailProfile = emailProfile;
    }
    
    public String getEmailProfile(){
        return emailProfile;
    }
    
    public void setIdProfile(String idProfile){
        this.idProfile= idProfile;
    }
    
    public void setBmr(int bmr){
        this.bmr = bmr;
    }
    
    public void setCalRequis(int calRequis){
        this.calRequis = calRequis;
    }
    
    public void setFatsRequis(int fatsRequis){
        this.fatsRequis = fatsRequis;
    }
    
    public void setCarbsRequis(int carbsRequis){
        this.carbsRequis = carbsRequis;
    }
    
    public void setProteinsRequis(int proteinsRequis){
        this.proteinsRequis = proteinsRequis;
    }
    
    public String getIdProfile(){
        return idProfile;
    }
    
    public int getBmr(){
        return bmr;
    }
    
    public int getCalRequis(){
        return calRequis;
    }
    
    public int getFatsRequis(){
        return fatsRequis;
    }
    
    public int getCarbsRequis(){
        return carbsRequis;
    }
    
    public int getProteinsRequis(){
        return proteinsRequis;
    }
    
    @Override
    public String toString(){
        return "\n" + "\nIdProfile : " + idProfile + "\nBMR : " + bmr + "\nCalRequis : " + calRequis + "\nFatsRequis : " + fatsRequis + "\nCarbsRequis : " + carbsRequis + "\nProteinsRequis : " + proteinsRequis + "\n";
    }
    
}
